shared/src/commonMain/kotlin/edu/usf/cutr/otp/serverinfo/model/ServerVersion.kt

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
We need unit tests for each API in the library, and example JSON response, to ensure it parses correctly.

You could do live parsing tests by pushing JSON responses to GitHub in the test resources folder and then querying GitHub as if it were the OTP server. Or figure out a way to give the test a local JSON file to parse. If pushing files to GitHub, let's push the test files in a separate PR so we can merge and fully test this PR using those live files before merging this PR.

Reply:
That sounds good! I think the live parsing tests sound like a good idea!

Reply:
@BumbleFlash Unless you already have them implemented, let's do the unit tests in a new PR so we can get this merged to master sooner. So I believe the remaining items before merge on this PR are Kotlin docs and changing the app ID.